dds-sweeper/dds-sweeper.c

```diff
@@ -832,6 +837,8 @@ void loop() {
         fast_serial_printf("%s\n", VERSION);
     } else if (strncmp(readstring, "status", 6) == 0) {
         fast_serial_printf("%d\n", local_status);
+    } else if (strncmp(readstring, "clkstatus", 9) == 0) {
+        fast_serial_printf("%d %lf %d", clk_mode, ad9959.ref_clk, ad9959.pll_mult);
```

Comment:
I think a newline is needed at the end of this print.

Reply:
One day I will remember the newline on every printf